server.js

```diff
@@ -0,0 +1,30 @@
+var express = require('express');
+var app = express();
+
+app.set('view engine', 'pug');
+
+var data = {
```

Comment:
maybe a comment here just to let it be known that this is the static data used for the grocery list display